### ISSUE TITLE
Feature: Dataset pre-processing scripts

### ISSUE THREAD

**Issue by lioutasb** [zcyang/imageqa-san#1]
Hello, I would like to implement this network on Torch but I'm struggling a bit so I thought to train at first with the implementation that you guys are uploaded here. Because I don't have much experience with theano and also I have a limited time I would really apprecated if you guys uploaded the pre-processing scripts for both image feature extraction and dataset splits or at least the trainval_feat.h5 and train/val1/val2.pkl files.

**Comment by zcyang** [comment]
The data files are very large, I will put them on google drive so they can be downloaded very soon. stayed tuned.

**Comment by zcyang** [comment]
refer to https://github.com/zcyang/imageqa-san#usage to download data

**Comment by lioutasb** [comment]
Thank you very much!

**Comment by ghost** [comment]
@zcyang 
Hi, I'm trying to reproduce the results from scratch, but it's hard to get the reported scores...
Could you share some details about the feature extraction? e.g. the CNN architecture (VGG-16 or 19), any normalization technique, etc.

**Comment by zcyang** [comment]
VGG-19 is used and no special technique is used.

**Comment by ghost** [comment]
Thank you!

**Comment by lioutasb** [comment]
It would be nice also if you could upload the pre-trained model used in paper.
